src/App.jsx

```diff
@@ -1,13 +1,16 @@
-import { useState } from 'react'
+import * as React from 'react';
 import './App.css'
 import TodoList from './TodoList'
 import AddTodoForm from './AddTodoForm'
 
 function App() {
+  const [newTodo, setNewTodo] = React.useState(''); 
```

Comment:
This is correct, well done. 

If you'd like to keep your code cleaner, I'd recommend the destructured version:

```
import { useState } from 'react

function App() {
    const [newTodo, setNewTodo] = useState('')
}

```